src/chocolatey/infrastructure.app/services/ChocolateyConfigSettingsService.cs

```diff
@@ -55,14 +55,14 @@ public IEnumerable<ChocolateySource> source_list(ChocolateyConfiguration configu
                         source.Id,
                         source.Disabled ? " [Disabled]" : string.Empty,
                         source.Value,
-                        string.IsNullOrWhiteSpace(source.UserName) ? string.Empty : "(Authenticated)",
+                        string.IsNullOrWhiteSpace(source.UserName) && string.IsNullOrWhiteSpace(source.Certificate) ? string.Empty : "(Authenticated)",

```

Comment:
I thing username is what authenticates, not the password... Or do you mean IsNullOrEmpty instead of WhiteSpace? Just sticking to what was here before to avoid any regressions.